### PR TITLE
Security alerts: Update  minimist --> ^1.2.4 and elliptic -> ^6.5.2

### PR DESCRIPTION
**Pull request by ghost**
Update  minimist --> ^1.2.4 and elliptic -> ^6.5.2 due to github security alerts and snyk alerts

Pull requests into Bonnie require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [ ] This pull request describes why these changes were made.
- [ ] This PR is into the correct branch.
- [ ] JIRA ticket for this PR:
- [ ] JIRA ticket links to this PR
- [ ] Code diff has been done and been reviewed (it **does not** contain: additional white space, not applicable code changes, debug statements, etc.)
- [ ] If UI changes have been made, google WAVE plug-in has been executed to ensure no 508 issues were introduced.
- [ ] Tests are included and test edge cases
- [ ] Tests have been run locally and pass (remember to update Gemfile when applicable)
- [ ] Test fixtures updated and documented as necessary ( see [internal wiki](https://gitlab.mitre.org/bonnie/internal-documentation/wikis/testing#test-fixtures) )
- [ ] Code coverage has not gone down and all code touched or added is covered. 
     * In rare situations, this may not be possible or applicable to a PR. In those situations:
         1. Note why this could not be done or is not applicable here: 
         2. Add TODOs in the code noting that it requires a test
         3. Add a JIRA task to add the test and link it here: 
- [ ] Automated regression test(s) pass

If JIRA tests were used to supplement or replace automated tests:
- [ ] JIRA test links:
- [ ] Justification for using JIRA tests:
- [ ] JIRA tests have been added to sprint

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [ ] JIRA tests have been run and pass
- [ ] You agree with the justification for use of JIRA tests or have provided input on why you disagree

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [ ] JIRA tests have been run and pass
- [ ] You agree with the justification for use of JIRA tests or have provided input on why you disagree

### CODE REVIEW

**Comment by ghost** [comment]
**bundle exec rake teaspoon**
Finished in 111.17200 seconds
387 examples, 0 failures, 19 pending

**bundle exec rake test**
Finished in 126.142136s, 1.4190 runs/s, 7.7611 assertions/s.
179 runs, 979 assertions, 0 failures, 0 errors, 8 skips

**Review by adongare — APPROVED** [review]
Looks good to me!

File: package.json [hunk 1]
```diff
@@ -109,5 +109,9 @@
   "dependencies": {
     "browserify": "^16.2.3",
     "cqm-execution": "^3.0.0"
+  },
```
Comment (AndrewBird81) [inline package.json:112]: Why are you using "resolutions"?

**Review by AndrewBird81 — COMMENTED** [review]
Why are you using "reso

**Pull request closed without merging**